### ISSUE TITLE
How to use log loss with xgboost in python?

### ISSUE THREAD

**Issue by avionrx** [dmlc/xgboost#1250]
param_test7 = {
 'n_estimators':[20]
}

clf = GridSearchCV(xgb.XGBClassifier(learning_rate =0.1,
                                 n_estimators=250,
                                 max_depth=9,
                                 min_child_weight=1,
                                 gamma=0.1,
                                 subsample=0.85,
                                 colsample_bytree=0.75,
                                 objective= 'multi:softprob',
                                 nthread=4,
                                 scale_pos_weight=1,
                                 seed=27,
                                 reg_alpha=0.01),
               param_test7, 
               verbose=1, 
               cv=5, 
               scoring='log_loss',
               n_jobs=4,
               iid=False)

 clf.fit(train_market_sample[non_leaky_predictors],train_market_sample["hotel_cluster"])

**Comment by pommedeterresautee** [comment]
Check documentation.
